src/background.js

```diff
@@ -180,33 +184,33 @@ async function getDirectories (src) {
 		var rej_p = rej;
         
         // THIS CODE ACTUALLY LOOKS FOR A PACKAGE.JSON TO REGISTER
-		glob(src + '/*/launcher_settings/interface_package.json', function(err, folders) {
-			console.log(folders);
-			if(folders) {
-				folders.forEach(function (folder) {
-					var folderToReturn = folder.replace("launcher_settings/interface_package.json", "");
-					interfacesToReturn.push(folderToReturn);
-				});
-				res_p();
-			} else {
-				rej_p("Failed to load directories.");
-			}
-		});
-		
-        // THIS CODE ONLY LOOKS FOR INTERFACE.EXE
-        
-		// glob(src + '/*/interface.exe', function(err, folders) {
+		// glob(src + '/*/launcher_settings/interface_package.json', function(err, folders) {
```

Comment:
Remove commented-out code?

Reply:
This code is here for if/when different methods of finding information is needed.